checkfiles/checkfiles.py

```diff
@@ -1019,15 +1030,18 @@ def run(out, err, url, username, password, encValData, mirror, search_query, fil
         if not dry_run:
             patch_file(session, url, job)
 
+        if not job.get('skip'):
```

Comment:
I think you need out.flush() and err.flush() after each file.   This will prevent the logged output from buffering.

Reply:
DONE